packages/components/src/index.ts

```diff
@@ -71,3 +71,4 @@ export { ExperienceControl } from './experience-control';
 export { default as SummaryButton } from './summary-button';
 export { default as CoreBadge } from './core-badge';
 export { default as Menu } from './menu';
+export { Tabs } from './tabs';
```

Comment:
I noticed this list is commented as "Logos" so we could reorganise a bit while here:

https://github.com/Automattic/wp-calypso/blob/cd358659ccc10695337c1d8f6d0c61dd0ee1ee4e/packages/components/src/index.ts#L60-L74

Reply:
Nice find, I moved exports to the right section as part of this PR.